backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -388,7 +388,7 @@ public VaultDto get(@PathParam("vaultId") UUID vaultId) {
 
 	@PUT
 	@Path("/{vaultId}")
-	@RolesAllowed("user")
+	@RolesAllowed("create-vaults")
```

Comment:
With this new role, we need to distinguish between create and update, otherwise an owner without the `create-vaults` role will not be able to update a vault:

![image](https://github.com/user-attachments/assets/add956d4-77b7-460f-99ea-705ce70aa851)

Reply:
This raises the question whether a vault owner without the create-vaults role is a data inconsistency? From a client perspective, is the intention of the feature request to block "creating" vaults or "owning" vaults? Maybe the name of role is misleading?

Reply:
addressed in 508c651